src/main/kotlin/uk/gov/justice/digital/hmpps/healthandmedication/jpa/FieldHistory.kt

```diff
@@ -19,7 +19,7 @@ import java.time.ZonedDateTime
 class FieldHistory(
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  val fieldHistoryId: Long = -1,
+  val fieldHistoryId: Long = 0,
```

Comment:
JPA changed how it was handling unpersisted entities... suddenly you're not allowed to have a default value of `-1` anymore to identify something not persisted 🤷